index.html

```diff
@@ -57,6 +60,164 @@ <h1 id="primary-text">
+                    </div>
+                </div>
+                <div class="work-card">
+                    <div class="bottom-area">
+                        <div class="top">
+                            <h1 class="text">
+                                Multi-post stories<br>
+                                Gain+Glory
+                            </h1>
+                        </div>
+                        <ul class="tech">
+                            <li class="tech-card">
+                                <p>Ruby on Rails</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>CSS</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>JavaScript</p>
+                            <li class="tech-card">
+                                <p>HTML</p>
+                            </li>
+                        </ul>
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section id="about-me">
+            <div id="ab-me-img"></div>
+            <div id="ab-me-1-img"></div>
+            <div id="ab-me-desktop-img"></div>
+            <div id="ab-me-desktop-1-img"></div>
+
+            <h1 class="title">About me</h1>
+            <p class="text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need coded,
+                don’t hesitate to contact me.
+            </p>
+            <button class="see-btn">
+                <p class=".text">
+                    Get my resume
+                </p>
+            </button>
+        </section>
+
+        <section id="skills">
+            <div class="skill-card">
+                <div class="skill-icon-ctn">
+                    <img class="skill-icon" src="icons/languages.svg" />
+                </div>
+                <p class="skill-text">Languages</p>
+                <ul class="skill-list">
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
+            <div class="skill-card">
+                <div class="skill-icon-ctn">
+                    <img class="skill-icon" src="icons/frameworks.svg" />
+                </div>
+                <p class="skill-text">Frameworks</p>
+                <ul class="skill-list">
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
```

Comment:
- [ ] Please make sure you are putting` frameworks` in the `Frameworks` category like React, Ruby on Rails, RSpec etc,  as mentioned in the design

|Figma design|Your design|
|----|-----|
|<img width="558" alt="Screenshot 2023-06-05 at 11 06 31 PM" src="https://github.com/Movgto/my-portfolio/assets/44798044/7c6df605-98d0-4350-bf1f-4a32764839f5">|<img width="307" alt="Screenshot 2023-06-05 at 11 06 47 PM" src="https://github.com/Movgto/my-portfolio/assets/44798044/d310bbb0-57c9-4e0a-9d82-2da109252a0d">|